UIKIT_DZ_Clone_AppStore/UIKIT_DZ_Clone_AppStore/ViewControllers/WebViewController/WebViewController.swift

```diff
@@ -0,0 +1,158 @@
+//
+//  WebViewController.swift
+//  UIKIT_DZ_Clone_AppStore
+//
+//  Created by Alexander Kokh on 13.07.2021.
+//
+
+import UIKit
+import WebKit
+
+final class WebViewController: UIViewController {
+
+    // MARK: - Public Properties
+    var productId = String()
+    var isPdf = false
+
+    // MARK: - Visual Components
+    private var webView = WKWebView()
+    private var toolBar = UIToolbar()
+    private var progressView = UIProgressView()
+
+    // MARK: - Private Properties
+    private let apiHost = "https://www.apple.com/ru/shop/product/"
+
+    // MARK: - UIViewController
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+    }
+
+    override func viewWillAppear(_ animated: Bool) {
+        webView.addObserver(self, forKeyPath: #keyPath(WKWebView.estimatedProgress), options: .new, context: nil)
+    }
+
+    override func viewWillDisappear(_ animated: Bool) {
+        webView.removeObserver(self, forKeyPath: #keyPath(WKWebView.estimatedProgress))//"estimatedProgress")
+    }
+
+    override func loadView() {
+        let webconfiguration = WKWebViewConfiguration()
+        webView = WKWebView(frame: .zero, configuration: webconfiguration)
+        view = webView
+    }
+
+    override func observeValue(forKeyPath keyPath: String?, of object: Any?, change: [NSKeyValueChangeKey : Any]?,
+                               context: UnsafeMutableRawPointer?) {
+        if keyPath == "estimatedProgress" {
+            progressView.progress = Float(webView.estimatedProgress)
+            if progressView.progress == 1 {
+                progressView.isHidden = true
+            } else {
+                progressView.isHidden = false
+            }
+        }
+    }
+
+    // MARK: - Public methods
+    @objc private func BackButtonPressed(button: UIBarButtonItem) {
+        if webView.canGoBack {
+            webView.goBack()
+        }
+    }
+
+    @objc private func ForwardButtonPressed(button: UIBarButtonItem) {
```

Comment:
```suggestion
    @objc private func forwardButtonPressed(button: UIBarButtonItem) {
```
еще c большой буквы?? это еще 10 отжиманий!